backend/types.ts

```diff
@@ -1,5 +1,24 @@
 export type Role = "User" | "Admin";
 
```

Comment:
Don't think all these should exist here _and_  in `backend/types/contentBlockType.ts`?

Reply:
Are they? Did Molly fix this? I don't see a `backend/types/contentBlockType.ts` file in this PR

Reply:
Hmm that was probably an older version ^

Reply:
Yeah I probably commented on the wrong line